tests/test_output.py

```diff
@@ -1310,22 +1307,130 @@ def test_wandb_logging_enabled(self, mock_wandb_init):
         config_path = fill_in_config_options(config_template_path, values_to_fill_dict, "wandb")
         mock_wandb_run = Mock()
         mock_wandb_init.return_value = mock_wandb_run
+
+        mock_summary = MagicMock()
+        mock_summary.__setitem__ = Mock()
```

Comment:
What's going on here?

Reply:
wand_run.summary is a dict which we access directly to log metrics. it needs to be mocked as well as it's setitem method, otherwise it throws an error (unlike a mock regular method that automatically returns a mock).

Reply:
Ah, okay. Can we add your explanation as a code comment?